packages/workbox-cli/src/lib/questions/ask-extensions-to-cache.ts

```diff
@@ -28,28 +28,24 @@ async function getAllFileExtensions(globDirectory: string) {
   const files: string[] = await new Promise((resolve, reject) => {
     // Use a pattern to match any file that contains a '.', since that signifies
     // the presence of a file extension.
-    glob(
-      '**/*.*',
-      {
-        cwd: globDirectory,
-        nodir: true,
-        ignore: [
-          ...constants.ignoredDirectories.map(
-            (directory) => `**/${directory}/**`,
-          ),
-          ...constants.ignoredFileExtensions.map(
-            (extension) => `**/*.${extension}`,
-          ),
-        ],
-      },
-      (error, files) => {
-        if (error) {
-          reject(error);
-        } else {
-          resolve(files);
-        }
-      },
-    );
+    glob('**/*.*', {
+      cwd: globDirectory,
+      nodir: true,
+      ignore: [
+        ...constants.ignoredDirectories.map(
+          (directory) => `**/${directory}/**`,
+        ),
+        ...constants.ignoredFileExtensions.map(
+          (extension) => `**/*.${extension}`,
+        ),
+      ],
+    })
+      .then((directories) => {
+        resolve(directories);
```

Comment:
Why is this `directories` now instead of `files`?

```suggestion
      .then((files) => {
        resolve(files);
```